src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends FigureColor {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
```

Comment:
Creating a square, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.